README.md

```diff
@@ -1,3 +1,94 @@
 # Argyle Android SDK example project
+Android Link SDK provides a way to integrate [Argyle Link](https://argyle.io/docs/argyle-link) into your Android app.
 
-Documentation can be found [here](https://argyle.io/docs/argyle-link/android-sdk-integration)
+If you are looking to update Argyle Link to the newest version, navigate to [upgrade guide](https://github.com/argyle-systems/argyle-link-android/blob/master/UPGRADING.md).
+
+**The SDK supports API level 23 and above ([distribution stats](https://developer.android.com/about/dashboards)).**
+
+**Note:** We recommend you to lock your app to portrait orientation.
+
+---
+- **Important:** When using tools like Proguard to obfuscate your code, make sure to exclude Android Link SDK package (`com.argyle.*`) from the process, it may cause unexpected runtime issues otherwise. You can do this by adding this line to your `proguard-rules.pro:-keep class com.argyle. { *; }`
+  
+Our target configuration is currently set to the following:
+
+- `minSdkVersion = 23`
+- `compileSdkVersion = 30`
+- `targetSdkVersion = 30`
+- `Android Support Library = 28.0.0`
+- `Kotlin = 1.4.32`
+
+### 1. Adding the SDK dependency
+```
+dependencies {
+    implementation 'com.argyle:argyle-plugin-android-source:3.x.x'
+}
+```
+
+### 2. Creating the SDK configuration
+```kotlin
+val config = ArgyleConfig.Builder()
+    .loginWith("YOUR_LINK_KEY", "https://api-sandbox.argyle.com/v1", "USER_TOKEN")
```

Comment:
I think we should definitely something in these readme's about what a `USER_TOKEN` is, and how integrators should use them

Reply:
We could link to the [user token guide](https://argyle.com/docs/products/returning-users-experience) we have in our docs